### ISSUE TITLE
Sticky- for Edge browser sticky is bumpy

### ISSUE THREAD

**Issue by tharshada** [OfficeDev/office-ui-fabric-react#7655]
### Environment Information 
 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
 https://fabricweb.z5.web.core.windows.net/oufr/6.49.0/#/examples/scrollablepane

 
#### Actual behavior:
[Edge](https://microsoft-my.sharepoint.com/:v:/p/hatrived/ES0BaKDgNYFOnt5ejvlGo7oBhsw6plk94R4ArcrG33iaNw?e=0cUfsA)

 
![image](https://user-images.githubusercontent.com/41147089/51202073-dafbde00-1923-11e9-8f56-325dfd89ab36.png)

![image](https://user-images.githubusercontent.com/41147089/51202224-447bec80-1924-11e9-8c36-d157e4d2ffba.png)

#### Expected behavior:
 [Chrome](https://microsoft-my.sharepoint.com/:v:/p/hatrived/EckWhVqxwa9AjVIk7-ZjhnABqM4RsBEbjtol36TecJY2uQ?e=1g6Epe)

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: Blocking
 
Products/sites affected: (if applicable)

**Comment by tharshada** [comment]
@leddie24 @mikewheaton @Vitalius1 can this be prioritized?

**Comment by Vitalius1** [comment]
I believe this could happen due to this line:
https://github.com/OfficeDev/office-ui-fabric-react/blob/ad66a786355b332c854d9e6c6f2afe43068f6236/packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx#L55.
Removing that debounce could cause some other performance issues.

**Comment by tharshada** [comment]
@shagra-ms

**Comment by leddie24** [comment]
I'm guessing it's because of the throttle also.  I know Edge does handle scroll event differently than Chrome and Firefox though, I've tried adding parallax to images and the scroll events in Edge produce a choppier effect when compared to Chrome/Firefox.  I want to say this is at the browser level.  I included links below demonstrating the issue with setting throttle to 1.  Even at a super slow scrolling rate, the problem repros in Edge, but not in Chrome and Firefox.

Edge:
https://streamable.com/jb34n

Firefox:
https://streamable.com/yxe8q

Chrome:
https://streamable.com/e90m0

**Comment by Jahnp** [comment]
Thanks for the insight Eddie. Sounds like it might be good for us to confirm with the Edge team if this is expected behavior for scroll handling, if there's a planned fix, why identical code produces different results, etc. As you pointed out, when we implemented parallax animations in SharePoint, we ended up having to disable them in Edge due to this.

Let's commit to figuring out a path here in February.
